tests/foreman/ui/test_host.py

```diff
@@ -865,6 +865,39 @@ def test_positive_search_by_parameter_with_operator(self):
             self.assertIsNotNone(self.hosts.wait_until_element(
                 (strategy, value % additional_host.name)))
 
+    @run_only_on('sat')
+    @tier2
+    def test_positive_search_with_org_and_loc_context(self):
+        """Perform usual search for host, but organization and location used
+        for host create procedure should have 'All capsules' checkbox selected
+
+        :id: 2ce50df0-2b30-42cc-a40b-0e1f4fde3c6f
+
+        :expectedresults: Search functionality works as expected and correct
+            result is returned
+
+        :BZ: 1405496
+
+        :CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        loc = entities.Location().create()
+        host = entities.Host(organization=org, location=loc).create()
+        with Session(self.browser) as session:
+            self.org.search_and_click(org.name)
+            self.org.click(tab_locators['context.tab_capsules'])
+            self.org.assign_value(locators['org.all_capsules'], True)
+            self.org.click(common_locators['submit'])
+            self.location.search_and_click(loc.name)
+            self.location.click(tab_locators['context.tab_capsules'])
+            self.location.assign_value(locators['location.all_capsules'], True)
+            self.location.click(common_locators['submit'])
```

Comment:
It will fit `_configure_org` quite normally (imo) as another check - we do have steps different from using `_configure_entity` in other modules (quick example - operatingsys), i'd rather say `_configure_entity` is not covering all our needs and we may want to extend it in future, but that's out of scope of current PR, of course.
Regarding parameters list - i do agree, it's quite ugly, but afaik we have an issue to replace huge parameters list with your approach, so it shouldn't be taken into consideration.
My point was - that's basically one of options for loc/org entities, and we may want to set it in future tests too (especially as there're customers BZ and we don't have such coverage), setting them 'manually' in each test doesn't seem like acceptable option for me, that's definitely should be moved away from test to some helper.